### ISSUE TITLE
Ideia/Request

### ISSUE THREAD

**Issue by marcosalfieri** [ArachisH/Tanji#2]
Is it possible to add a tool that you can attribute a hotkey to the "Send to server" button instead of having to click the button itself everytime I have to send an specific packet? Thanks!!!

**Comment by ArachisH** [comment]
Oooh, like assigning a key to a packet?
[0][0][0][1] = F1
[0][0][0][2] = F2

Like that?; Or do you only want to hit the (Send To X) when a key is pressed?

**Comment by marcosalfieri** [comment]
Yes, like you said! When you press F1, the packet would be sent (even if Tanji is minimized) and you wouldn't need to maximize Tanji and click the "Send to Server" button. This would be extremely very useful in case you want to send a packet and can't lose time opening Tanji and clicking the button

**Comment by ArachisH** [comment]
This seems really useful  now that I think about it, I'll try to add it in before the next update.

**Comment by marcosalfieri** [comment]
Thanks! =)

**Comment by Prokprok** [comment]
What happen to the rest of the options? i mean i only see Sheduler.. what happen to toolbox etc..?

**Comment by Speaqer** [comment]
It's only for developers at the moment so all features are not coded yet.

**Comment by Prokprok** [comment]
The idea is good but.. cant even send packets.. so what is the purpose of only log packets xd

**Comment by ArachisH** [comment]
It was currently released so people can play around with the new extension system, tools will soon be added as a fine-tune some more things.

**Comment by ArachisH** [comment]
I've decided to add this feature once the injection tools that v0.10 are present in v1.X.

**Comment by HbbJ** [comment]
Hi Arachis I wonder if u could add the connection to Hevvohotel.us, I tried with other retros and i can connect but with that i can't it

**Comment by Mort2rire** [comment]
Hello, Can you try to decrypt "Hlove.fr" & "Wibbo.fr". Thanks. 

Mort²rire

**Comment by ArachisH** [comment]
Don't spam this post with irrelevant questions to the main issue. Turn to issue #5 for more information on more retro support.

**Comment by ArachisH** [comment]
There is a pre-release build(https://github.com/ArachisH/Tanji/releases/tag/v1.4.380.0) that offers this functionality in the Scheduler page. You can set the schedule's Cycle property to '1' so that the packet is only sent once before it toggles itself off, otherwise leave it at '0' for continuous sending.
